clients/admin-ui/src/features/config-wizard/setup.tsx

```diff
@@ -36,7 +36,7 @@ const Setup = () => {
           width="50%"
         >
           <Heading as="h3" size="lg">
-            Get Started
+            Generate your DataMap
```

Comment:
Should this be Data Map?

Reply:
The text was copied directly from the issue

Reply:
The figma has a space though, I think it was a copy probly.